scripts/update-framework-results.py

```diff
@@ -23,21 +23,23 @@
 
 from dmscripts.env import get_api_endpoint_from_stage
 from dmscripts.logging import configure_logger
-from dmscripts.insert_dos_framework_results import insert_result
```

Comment:
A bunch of constants from that file are used in another script, but the code is unused.

It does, however, capture the more complex logic we had for DOS assessment - with the lot-specific "declaration"  questions that led to the logic of individual services/lots being able to fail, which we don't have for G-Cloud at all.

I don't know if this is something that we want to keep around for posterity in case we have that kind of thing again in future (it is a bit fiddly), or if we just want to clear it out in an effort to streamline our scripts a bit.